.github/workflows/build-and-test.yml

```
@@ -58,9 +58,70 @@ jobs:
+        directory: ./TestResults
+        fail_ci_if_error: false
+        verbose: true
+
   build-artifacts:
-    runs-on: windows-latest
+    runs-on: ${{ matrix.os }}
     if: github.event_name == 'workflow_dispatch'
+    strategy:
+      matrix:
+        include:
+          - os: windows-latest
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)